releasenotes/notes/fixwinprocesshandleshang-bce162f5c0fee73f.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    On Windows, fixes problem in process agent wherein windows processes
```

Comment:
```suggestion
    On Windows, fixes problem in process agent wherein Windows processes
```

Reply:
I did a wrong merge and this change is not part of my development.